_projects/2024/stingray/dashboard.md

```diff
@@ -0,0 +1,72 @@
+---
+name: A quicklook dashboard for X-ray astronomy with Stingray
+desc: Create a dashboard that displays some quicklook analysis of astronomical X-ray observations.
+requirements:
+ - Experience with Python
+difficulty: medium
+issues:
+ - https://github.com/StingraySoftware/stingray/issues/794
+mentors:
+ - matteobachetti
```

Comment:
will need someone else to be named as a secondary mentor here (enough to come up with the name once we start)